app/src/main/java/pl/futurecollars/invoicing/model/TaxCalculatorResult.java

```diff
@@ -0,0 +1,23 @@
+package pl.futurecollars.invoicing.model;
```

Comment:
ta klasa jest raczej modelem na poziomie wymiany danych, a nie modelem bazodanowym, więc oddzielmy go od tych z pakietu model. Po ang taka klasa to DTO, data transfer object i może być w pakiecie controllera, albo service